linkerd/protocol/http/src/test/scala/io/buoyant/linkerd/protocol/HttpInitializerTest.scala

```diff
@@ -139,12 +140,13 @@ class HttpInitializerTest extends FunSuite with Awaits with Eventually {
     val maxInitLineSize = hparam.MaxInitialLineSize(30.kilobytes)
     val maxReqSize = hparam.MaxRequestSize(40.kilobytes)
     val maxRspSize = hparam.MaxResponseSize(50.kilobytes)
+    val maxErrRspSize = Headers.param.MaxErrResponseSize(30.kilobytes)
     val streaming = hparam.Streaming(false)
     val compression = hparam.CompressionLevel(3)
 
     val router = HttpInitializer.router
       .configured(maxHeaderSize).configured(maxInitLineSize)
-      .configured(maxReqSize).configured(maxRspSize)
+      .configured(maxReqSize).configured(maxRspSize).configured(maxErrRspSize)
```

Comment:
Maybe add a newline for continuity/readability.

Reply:
done